### PR TITLE
Fix inconsistent handling of user information in issue processing and author post-processing

### PR DESCRIPTION
**Pull request by bockthom**
In this PR, two problems regarding inconsistent handling of user information are fixed:

1. In issue processing (and jira issue processing), the user buffer to reduce the amount of database queries is now updated correctly before re-assigning the disambiguated user data to the event data.

2. In author post-processing, the treatment of the `event.info.2` column is adjusted as the column's entries are surrounded by quotation marks.

### CODE REVIEW

File: issue_processing/jira_issue_processing.py [hunk 1]
```diff
@@ -513,37 +515,64 @@ def get_or_update_user(user, buffer_db=user_buffer):
...
         # check database for issue author
-        issue["author"] = get_or_update_user(issue["author"])
+        issue["author"] = get_id_and_update_user(issue["author"])
 
         # check database for event authors
         for comment in issue["comments"]:
-            # get the event user from the DB
-            comment["author"] = get_or_update_user(comment["author"])
-            ## get the reference-target user from the DB if needed
+            comment["author"] = get_id_and_update_user(comment["author"])
```
Comment (hechtlC) [inline issue_processing/jira_issue_processing.py:560]: In the following couple of lines you are a little inconsistent with the "#" signs for the comments. Sometimes you use one sometimes two

File: issue_processing/jira_issue_processing.py [hunk 2]
```diff
@@ -513,37 +515,64 @@ def get_or_update_user(user, buffer_db=user_buffer):
...
+        # get issue author
+        issue["author"] = get_user_from_id(issue["author"])
+
+        # get event authors
+        for comment in issue["comments"]:
+            comment["author"] = get_user_from_id(comment["author"])
+            ## get the reference-target user if needed
             # if event["ref_target"] != "":
-            #   event["ref_target"] = get_or_update_user(event["ref_target"])
+            #     event["ref_target"] = get_user_from_id(event["ref_target"])
```
Comment (hechtlC) [inline issue_processing/jira_issue_processing.py:575]: Why the extra space?

**Review by hechtlC — APPROVED** [review]
Except for the small inconsistencies in the comments I think this looks really good. Thanks @bockthom !

**Review by clhunsen — APPROVED** [review]
Looks good to me. Nice work, @bockthom!

**Pull request merged**